### PR TITLE
New version: KristenMcWilliam.Nyrna version 2.11.1 [FP-O]

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Result: Installation Failed
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | Nyrna |     |
| Version     | 2.11.1     |  |
| Publisher   | Kristen McWilliam   |       |
| ProductCode |           |     |
#### Auto-updated by [vedantmgoyal2009/winget-pkgs-automation](https://github.com/vedantmgoyal2009/vedantmgoyal2009/tree/main/winget-pkgs-automation) in workflow run [5723](<https://github.com/vedantmgoyal2009/vedantmgoyal2009/actions/runs/4076810034>)

### CODE REVIEW

**Comment by stephengillie** [comment]
This package installs normally, but requires `Microsoft.VCRedist.2015+.x64` to run:
image

**Review by ItzLevvie — APPROVED** [review]

File: manifests/k/KristenMcWilliam/Nyrna/2.11.1/KristenMcWilliam.Nyrna.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: user
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2023-02-02
+Installers:
```
Comment (stephengillie) [inline manifests/k/KristenMcWilliam/Nyrna/2.11.1/KristenMcWilliam.Nyrna.installer.yaml:17]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**